locales/en/messages.json

```diff
@@ -6848,10 +6848,10 @@
     "firmwareFlasherBuildCustomDefines": {
         "message": "Custom Defines"
     },
-    "classicBuild": {
-        "message": "Classic Mode"
+    "coreBuild": {
+        "message": "Core (formally 'classic')"
```

Comment:
Core Mode ?

Reply:
I am not sure it is a "mode". If anything it would be `Core Only`

Reply:
Please abandon term classic